### PR TITLE
Ci/add ruff format part4 - the final format!

### PR DESCRIPTION
**Pull request by atravitz**
this is all the CLI tests, so I went with my personal preferences on any edge cases.

Checklist
* [ ] Added a ``news`` entry

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -50,7 +50,6 @@ fallback_version = "0.0.0"
 [tool.ruff]
 line-length = 100
 
-format.exclude = [ "openfecli/tests/*" ]
```
Comment (mikemhenry) [inline pyproject.toml:53]: 🎉

**Review by mikemhenry — APPROVED** [review]
